src/kiwixapp.cpp

```diff
@@ -24,6 +24,7 @@ KiwixApp::KiwixApp(int& argc, char *argv[])
         appendToDirectory(m_libraryDirectory.toStdString(),"library.xml"),
         m_settingsManager.getKiwixServerPort()))
 {
+    m_language.setTranslationsFromJsonFile(":/i18n/en.json");
```

Comment:
We should use the QLocale() to load the right json file.
And it would be better if we path the QLocale directly to the function and the function find itself what to load.